test/android/core/javatests/src/org/xwalk/core/xwview/test/ShouldOverrideUrlLoadingTest.java

```diff
@@ -469,7 +471,7 @@ public Boolean call() throws Exception {
             }
         });
 
-        assertEquals(1, mShouldOverrideUrlLoadingHelper.getCallCount());
+        assertEquals(shouldOverrideUrlLoadingCallCount, mShouldOverrideUrlLoadingHelper.getCallCount());
```

Comment:
I think it's easier to understand what's being tested here if you just use `0` instead. This is 0, right?